### ISSUE TITLE
How to fine tune DiT on custom data for Object Detection task?

### ISSUE THREAD

**Issue by deshwalmahesh** [thibaultvt/Diard#7]
Thanks for the code. There's one question from my side that given all the bounding boxes for the image, how can we fine tune `DiT` on objection detection task?

Let's suppose I want to predict 2 different types of BBoxes on my images, how can I do that? What would be the changes in config file, code etc? I don't have masks nor do I want to predict. Just the bounding boxes.

[I have asked a similar question here](https://stackoverflow.com/questions/73509188/how-to-fine-tune-pre-trained-transformers-backbone-dit-for-object-detection) 

Any blog, code, your code etc etc would be helpful.

thanks in advance :)

**Comment by thibaultvt** [comment]
The only documentation and code I could refer you to is the unilm/dit repository.

Excuse me for not being more helpful, hopefully you are able to figure it out.
